### DISCUSSION TITLE
I can't get "Admin panel accessible from" to work, help?

### DISCUSSION THREAD

**Discussion by pdegan2814** [rejetto/hfs#650] [Q&A]
I'm trying to limit admin panel access to just the IPs on my local network. I've tried "192.168.0.*", "192.168.0.0/24" and "192.168.0.1-192.168.0.254" but every time I set one of those values, I'm immediately blocked from the admin panel, even on localhost. I can log into the server with the admin account I created, but when I go to Options and then click on the button to enter the admin panel, I get "Forbidden". What am I doing wrong?

**Comment by rejetto** [comment]
if you set `192.168.0.*`, you cannot enter using localhost.
if you want to to include localhost you must set 192.168.0.*|::1|127.0.0.1
